core/src/main/java/org/apache/accumulo/core/clientImpl/OfflineIterator.java

```diff
@@ -284,7 +284,7 @@ private TabletMetadata getTabletFiles(Range nextRange) {
     // possible race condition here, if table is renamed
     String tableName = Tables.getTableName(context, tableId);
     AccumuloConfiguration acuTableConf =
-        new ConfigurationCopy(context.tableOperations().getProperties(tableName));
+        new ConfigurationCopy(context.tableOperations().getPropertiesMap(tableName));
```

Comment:
I have not made any changes to this part. I felt as though it might be better for someone with more experience with the use and nature of the ConfigurationCopy constructor to make the decision on this.